server/models/Collective.ts

```diff
@@ -3064,7 +3064,8 @@ class Collective extends Model<
   };
 
   getImageUrl = function (args = {}) {
-    return getCollectiveAvatarUrl(this.slug, this.type, this.image, args);
+    const slug = this.data?.originalOrganizationProps?.slug || this.slug;
```

Comment:
Not a fan of this:
1. What if the original slug is then used by another profile? It sounds like this will return another profile's picture
2. How do you then change the image or the vendor profile?